homeassistant/components/philips_js/config_flow.py

```diff
@@ -81,6 +74,38 @@ def __init__(self) -> None:
         self._hub: PhilipsTV | None = None
         self._pair_state: Any = None
 
+    async def _async_attempt_prepare(
+        self, host: str, api_version: int, secured_transport: bool
+    ) -> None:
+        hub = PhilipsTV(
+            host, api_version=api_version, secured_transport=secured_transport
+        )
+
+        await hub.getSystem()
+        await hub.setTransport(hub.secured_transport)
```

Comment:
Can this raise?

Reply:
In can raise some subclass of GenericFailure. All stemming from httpx connectivity failures.

Reply:
Want to catch this one layer out since there are two option to handling it. Either abort or error in form.